drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -123,7 +123,7 @@
         </div>
         <div class="row">
           <label for="item-{{id}}-numerical-margin">
-            {{i18n "Margin ± (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
+            {{i18n "Margin +/- (when a numerical value is required, values entered by students must not differ from the expected value by more than this margin; default is zero)"}}
```

Comment:
@itsjeyd Hmm, you'd think a method called `load_unicode` would support loading unicode text :-p

Your workaround is fine for now, so we can get this merged on time. But the proper fix is to change xblock-utils to use `return unicode(resource_content, 'utf-8')` instead of just `return unicode(resource_content)`. Would you be able to open a second PR to fix that in xblock-utils?